cluster/gce/manifests/kube-addon-manager.yaml

```diff
@@ -23,7 +23,7 @@ spec:
         - all
     # When updating version also bump it in:
     # - test/kubemark/resources/manifests/kube-addon-manager.yaml
-    image: k8s.gcr.io/addon-manager/kube-addon-manager:v9.1.4
+    image: k8s.gcr.io/addon-manager/kube-addon-manager:v9.1.5
```

Comment:
Look like this image is not pushed yet - we will need to promote the staging images to prod first: e.g. https://github.com/kubernetes/k8s.io/pull/1760

Reply:
@spencer-p Feel free to send a PR for that and assign it to me. Or I will do it when I get a chance :)

Reply:
If I understand correctly, I have to be a member/owner of the staging group or image repo - which looks to be [just you and jsand@google.com](https://github.com/kubernetes/k8s.io/blob/main/groups/groups.yaml#L664).

Reply:
Sorry I missed that! Well once you become an org member we can add you there. I opened https://github.com/kubernetes/k8s.io/pull/2002 to publish the images.